client/app/components/packages/landuse-form/show.hbs

```diff
@@ -656,7 +656,7 @@
                   Is the Action a modification to or pursuant to a Legal Instrument?
                 </A.Prompt>
                 <A.Field>
-                  {{optionset 'landuseAction' 'dcpIndicatewhetheractionisamodification' 'label' landuseAction.dcpDateofpreviousapproval}}
+                  {{optionset 'landuseAction' 'dcpIndicatewhetheractionisamodification' 'label' landuseAction.dcpIndicatewhetheractionisamodification}}
```

Comment:
I can't believe `dcpIndicatewhetheractionisamodification` is a real variable name, it may be the worst one yet.